CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
 
 ## Latest Changes
 
+## Version 2.0.1 @dataclass from attr bug and no more callable objects.
+
+We were using `@dataclass` decorator from attr, which  caused kaiba not to run when used in an environment that did not have attr installed. Thanks to @ChameleonTartu for finding and reporting the bug.
+We are now callable object free. From now we will only use normal functions. 
```

Comment:
@thomasborgen `normal` is opinion-based. It can be just `... we will only functions`